jest.config.js

```diff
@@ -0,0 +1,10 @@
+const commonCofig = require('@folio/stripes-acq-components/jest.config');
+
+module.exports = {
+  ...commonCofig,
+  testMatch: ['**/OrganizationSearch/?(*.)test.{js,jsx}'],
+  collectCoverageFrom: [
```

Comment:
you still need to define folder for coverage for sonar, now it's coverage-jest, but should be coverage